project/npda/migrations/0002_alter_npdauser_role.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 5.0.6 on 2024-06-30 18:36
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("npda", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="npdauser",
+            name="role",
+            field=models.PositiveSmallIntegerField(
+                choices=[
+                    (1, "Coordinator"),
```

Comment:
Maybe reference the constants in user.py here so we have the string role descriptions in one place?